packages/zod-openapi/package.json

```diff
@@ -52,7 +52,8 @@
   },
   "dependencies": {
     "@asteasolutions/zod-to-openapi": "^5.5.0",
-    "@hono/zod-validator": "0.1.11"
+    "@hono/zod-validator": "0.1.11",
+    "yaml": "^2.4.0"
```

Comment:
I don't want to include `yaml` in the dependencies as much as possible. So, it might be a good idea to specify `yaml` in `peerDependencies`.